.gitignore

```diff
@@ -1 +1,7 @@
 build/
+nodejs/index.js
+nodejs/node_modules/
```

Comment:
It's a precaution :P so they don't get added to the repo. It has happened before.